docs/openapi/api-private-v1.yaml

```diff
@@ -125,6 +172,57 @@ components:
           type: array
           items:
             $ref: './schemas-pn-timeline-v1.yaml#/components/schemas/TimelineElement'
+    
+
+    ############################################################################################
+    ###                 COMUNICAZIONE DELLA VISUALIZZAZIONE DI UNA NOTIFICA                  ###
+    ############################################################################################
+
+    RequestNotificationViewedDto:
+      type: object
+      required:
+        - iun
+        - recipientType
+        - recipientInternalId
+        - raddType
+        - raddBusinessTransactionId
+        - raddBusinessTransactionClosingDate
+      properties:
+        iun:
+          type: string
+          description: Identificativo Univoco della Notifica
+        recipientType:
+          type: string
+          description: |
+            tipo del destinatario <br/>
+            __PF__: Persona Fisica <br/>
+            __PG__: Persona Giuridica <br/>
+        recipientInternalId:
+          type: string
+          description: identificativo anonimizzato del destinatario
+        raddType:
```

Comment:
Metterei la definizione di questo tipo in un file comune e metterei un pattern di validazione

Reply:
Che genere di validazione?

Reply:
pattern, lunghezza

Reply:
ok riguardo questo bisogna chiedere a qualcuno perchè nella issue non è presente nessuna indicazione a riguardo